README.md

```diff
@@ -34,10 +34,20 @@ There are a number of configuration details defined in environment variables. Th
 
 - **GTFS_URL**: Location of the GTFS zip file
 - **BUSLOC_URL**: Source of GTFS-realtime enhanced VehiclePositions json data file
-- **SWIFTLY_REALTIME_VEHICLES_URL** and **SWIFTLY_AUTHORIZATION_KEY**: Source of Swiftly vehicle data
+- **SWIFTLY_REALTIME_VEHICLES_URL**: Source of Swiftly vehicle data
+- **SWIFTLY_AUTHORIZATION_KEY**: for dev only, see below for prod
 - **SKATE_HASTUS_URL**: Source of extended schedule data
 - **TRIP_UPDATES_URL**: Source of GTFS-realtime enhanced TripUpdates json data file (optional)
-- **SECRET_KEY_BASE** Used for writing encrypted cookies. Generate a value using `mix phx.gen.secret` (only required in production)
-- **ERLANG_COOKIE** Used by Erlang (only required in production)
+- **API_URL**: URL of the API for retrieving live train positions
+- **API_KEY**: Access key for the API (for dev only, see below for prod)
+- **ERLANG_COOKIE**: Used by Erlang (only required in production)
```

Comment:
I thought this changed to `RELEASE_COOKIE` or similar?

Reply:
It sure did, good catch.